### ISSUE TITLE
Conversion to mnn

### ISSUE THREAD

**Issue by alcaster** [wlguan/Stronger-yolo-pytorch#15]
How to convert to MNN ? I tried converting to onnx and then to mnn but I got this error.
` terminate called after throwing an instance of 'Error'
  what():  [11:04:10] /home/mchalecki/github_sources/MNN/tools/converter/source/onnx/SliceOnnx.cpp:84: Check failed: 1 == axis ==> Only support axis equal to 1`
In earlier version I saw the code to convert it to tensorflow first. What's the status, do you plan to implement it in near future ?

**Comment by wlguan** [comment]
If you want to convert to onnx, you should cut off the postprocess since the slice op is not well supported by pytorch->onnx. Then you can do the postprocess directly in cpp project. I will update the conversion script to tensorflow->MNN this month. :)

**Comment by alcaster** [comment]
Great, thank you for response :)

**Comment by wlguan** [comment]
@alcaster  MNN is supported. Contact me if have any question.

**Comment by imyhxy** [comment]
@wlguan Thanks for your great work! I try to port the model to MNN format. But the `port2mnn.py` script only saves the `*.pb` file. I used the `mnnconvert` tool to do that job but got a **Segmentation fault** error.
